app/src/test/java/com/codesoom/assignment/controllers/UserControllerTest.java

```diff
@@ -103,7 +118,9 @@ void updateUserWithValidAttributes() throws Exception {
         mockMvc.perform(
                 patch("/users/1")
                         .contentType(MediaType.APPLICATION_JSON)
-                        .content("{\"name\":\"TEST\",\"password\":\"test\"}")
+                        .content("{\"name\":\"TEST\",\"password\":\"TEST\"}")
+                        .header("Authorization", "Bearer" + MY_TOKEN)
```

Comment:
```suggestion
                        .header("Authorization", "Bearer " + MY_TOKEN)
```

여기에 띄어쓰기가 잘못되어있었던 것 같네요

Reply:
그렇다고 하더라도 왜 NullPointerException이 발생했고, 왜 테스트에서 실패했는지 쉽게 찾을 수 없었는지 고민해 볼 필요가 있습니다.